html/pfappserver/lib/pfappserver/Model/DB.pm

```diff
@@ -45,7 +45,7 @@ sub assign {
 
     # Create global PF user
     foreach my $host ("'%'","localhost") {
-        my $sql_query = "GRANT SELECT,INSERT,UPDATE,DELETE,EXECUTE,LOCK TABLES ON $db.* TO ?\@${host} IDENTIFIED BY ?";
+        my $sql_query = "GRANT SELECT,INSERT,UPDATE,DELETE,EXECUTE,LOCK TABLES, CREATE TEMPORARY TABLE ON $db.* TO ?\@${host} IDENTIFIED BY ?";
```

Comment:
Do we really need this grant? If so then it should be in the upgrade script for v10

Reply:
That was part of some possible changes needed for the aggregation.
I committed it by mistake. If in the final code it is not needed I will remove it.